src/Store/Reducer.js

```diff
@@ -83,27 +83,25 @@ export const reduser = (state, action) => {
           else delete errors[name]
           break
         case 'checkbox':
```

Comment:
try to just remove this case at all and if it's not work explain me please why

Reply:
Checkbox field errors are not being added to the errors object. They are handled specific way

Reply:
I see. Can we handle that case in validation function ?. Please try it somehow )